src/openforms/js/components/admin/form_design/registrations/objectsapi/ObjectsApiOptionsFormFields.js

```diff
@@ -59,6 +59,7 @@ const ObjectsApiOptionsFormFields = ({index, name, schema, formData, onChange})
           draft.geometryVariableKey = '';
         } else {
           delete draft.variablesMapping;
+          delete draft.geometryVariableKey;
```

Comment:
Not removing it when switching to v1 would result in a `ValidationError` in the backend

Reply:
Is being included in PR for #4567